EXPLAINER.md

```diff
@@ -451,34 +562,42 @@ increasing the [fingerprinting](https://w3c.github.io/fingerprinting-guidance)
 cross-screen placements to act in deceptive, abusive, or annoying manners.
 
 To help mitigate these concerns, user permission should be required for sites to
-get multi-screen information and place windows on other screens. Given the API
-shape proposed above, user agents could reasonably prompt users when sites call
-`getScreens()`, fulfilling the promise with requisite information for
-cross-screen placement requests if the user accepts the prompt, and rejecting
-the promise if the user denies access. If the permission is not already granted,
-cross-screen placement requests could fall back to same-screen placements,
-matching pre-existing behavior of some user agents. The amount of information
-exposed to a given site would be at the discretion of users and their agents,
-which may expose no new information, or subsets for more limited use cases.
-
-The `isMultiScreen()` method could fulfill its promise without a user prompt,
-exposing a minimal single bit of information to support some critical features
-(e.g. show/hide multi-screen entry points like “Show on another screen”), and to
-avoid unnecessarily prompting single-screen users for inapplicable information
-and capabilities. Similarly, `screenschange` events that change the result of
-`isMultiScreen()` queries could be fired without permission gates, to obviate
-the need for sites to poll that method; and that could be limited to sites that
-have previously called `isMultiScreen()`.
-
-User agents can measure and otherwise intervene when sites request the newly
-proposed information or use the newly proposed capabilities.
+access nontrivial multi-screen information and place windows on other screens.
+Given the API shape proposed above, user agents could reasonably prompt users
+when sites call `getScreens()`, fulfilling the promise if the user accepts the
+prompt, and rejecting the promise if the user denies access. If the permission
+is not already granted, cross-screen placement requests could fall back to
+same-screen placements, matching pre-existing behavior of some user agents. The
+amount of information exposed to a given site would be at the discretion of
+users and their agents.
+
+The `Screen.isExtended` boolean is exposed without explicit permission checks,
+as this minimal single bit of information supports some critical features for
+which a permission prompt would be obtrusive (e.g. show/hide multi-screen UI
+entry points like “Show on another screen”), and helps avoid unnecessarily
+prompting single-screen users for inapplicable information and capabilities.
+A point to note here is that many user agents already effectively expose the
+presence of multiple screens to windows located on secondary screens, where
+`window.screen.availLeft|Top` >> 0. Script access of this bit is a detectable
+[active fingerprinting](https://w3c.github.io/fingerprinting-guidance/#active)
+signal, which may be observed and blocked by the user agent.
```

Comment:
Might be worth noting that we're following TAG's design principles here.

https://w3ctag.github.io/design-principles/#device-enumeration

"When designing API which allows users to select a device, it may be necessary to also expose the fact that there are devices are available to be picked. This does expose one bit of fingerprinting data about the user’s environment to websites, so it isn’t quite as safe as an API which doesn’t have such a feature."

"The trade-off is that by allowing websites this extra bit of information, the API lets authors make their user interface less confusing. They can choose to show a button to trigger the picker only if at least one device is available."

Reply:
Done; thanks for the tip!